pipeline-model-declarative-agent/pom.xml

```diff
@@ -0,0 +1,137 @@
+  ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  ~ THE SOFTWARE.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>2.13</version>
+    <relativePath/>
+  </parent>
+
+  <groupId>org.jenkinsci.plugins</groupId>
+  <artifactId>pipeline-model-declarative-agent</artifactId>
+  <version>0.5-SNAPSHOT</version>
+  <packaging>hpi</packaging>
+  <name>Pipeline: Declarative Agent API</name>
+  <description>An opinionated, declarative Pipeline</description>
+  <url>https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Model+Definition+Plugin</url>
+
+  <licenses>
+    <license>
+        <name>MIT</name>
+        <url>http://opensource.org/licenses/MIT</url>
+        <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>abayer</id>
+      <name>Andrew Bayer</name>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git://git@github.com/jenkinsci/pipeline-config-plugin.git</connection>
+    <developerConnection>scm:git:git@github.com:jenkinsci/pipeline-config-plugin.git</developerConnection>
+    <url>https://github.com/jenkinsci/pipeline-config-plugin</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-cps</artifactId>
+      <version>2.18</version>
+    </dependency>
+  </dependencies>
+  
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.gmaven</groupId>
+        <artifactId>gmaven-plugin</artifactId>
+        <version>1.4</version>
+        <configuration>
+          <providerSelection>1.8</providerSelection>
+        </configuration>
+        <executions>
+          <execution>
+            <goals>
+              <goal>generateStubs</goal>
+              <goal>compile</goal>
+              <goal>generateTestStubs</goal>
+              <goal>testCompile</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <systemProperties>
+            <property>
+              <name>hudson.udp</name>
+              <value>33849</value>
+            </property>
+          </systemProperties>
+          <argLine>-Xmx4g -XX:MaxPermSize=256m</argLine>
+          <reuseForks>true</reuseForks>
+          <forkCount>0.5C</forkCount>
+          <!-- TODO: I hate the retrying tests but given the nondeterministic whackiness of JENKINS-37101... -->
+          <rerunFailingTestsCount>3</rerunFailingTestsCount>
```

Comment:
Why are you overriding this? It is on by default, and in a better way than you have configured. Best to delete this whole `plugin` section.